ChangeLog.md

```diff
@@ -341,6 +341,8 @@
 
   * Fix flaky test cases in `pl-number-input` which didn't handle comparison options properly (Mingjie Zhao).
 
+  * Fix bug in unicode encode/decode in file editor (Tim Bretl).
```

Comment:
Entry should be under the **__next version__** section.

Reply:
done